web/src/app/core/services/channel.service.ts

```diff
@@ -13,8 +13,13 @@ import { Injectable } from '@angular/core';
 import { Subject, Observable } from 'rxjs';
 import { filter, map } from 'rxjs/operators';
 
+export enum keyChannelStrim {
```

Comment:
cast the enum to a string enum https://www.typescriptlang.org/docs/handbook/enums.html#string-enums